requirements.txt

```diff
@@ -1,6 +1,6 @@
-torch==1.8.1
-hydra_core==1.0.6
-wandb==0.10.30
-transformers==4.6.0
-accelerate==0.3.0
-datasets==1.11.0
+torch==1.8.1  # 1.9 also works for Colab TPU, but not for GPU/CPU; the bugfix of 1.8.2 seems irrelevant for now.
+hydra_core>=1.1,<1.2  # pip will likely update it to 1.1.1, but it is probably ok and good for bugfixes.
+wandb>=0.10.32,<1  # pip will likely update it to 0.12.1, but it is probably ok and good for bugfixes.
+transformers>=4.6.0,<5  # pip will likely update it to 4.10.0, but it is probably ok and good for bugfixes.
+accelerate>=0.4.0,<1  # We may want to use 0.5.0 in the near future
+datasets[streaming]>=1.11.0,<2
```

Comment:
Same as https://github.com/bigscience-workshop/metadata/pull/19/files#r702606374
See https://python-poetry.org/docs/dependency-specification/ for translating Poetry's `^` to pip requirements.txt.